src/baldr/accessrestriction.cc

```diff
@@ -43,6 +61,37 @@ void AccessRestriction::set_value(const uint64_t v) {
   value_ = v;
 }
 
+const json::MapPtr AccessRestriction::json() const {
+  auto map = json::map({{"type", type_to_string[type()]},
```

Comment:
```suggestion
  auto maybe_found = type_to_string.find(type());
  std::string restriction_type = "unsupported";
  if (maybe_found != type_to_string.cend()) {
    restriction_type = maybe_found->second;
  }
  auto map = json::map({{"type", restriction_type},
```

Reply:
this will happen when we add a new type and old code is accessing it

Reply:
good point! thanks!